app/models/subscription.rb

```diff
@@ -74,6 +74,12 @@ def next_subscription
     next_subscriptions.not_canceled.order(created_at: :desc).first
   end
 
+  def fee_exists?(timestamp)
```

Comment:
Looks good, but I would just suggest to pass the date in here, rather than the timestamp :)